tests/unit/test_install.py

```diff
@@ -1886,3 +1886,25 @@ def test_save_config_ext_hms(ws, mock_installation):
             'num_days_submit_runs_history': 30,
         },
     )
+
+
+def test_upload_dependencies(ws, mock_installation):
```

Comment:
Also add an installation integration test. The one that tests for repair job is a good small example

Reply:
added int test